ajax/libs/jquery.printarea/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "jquery.printarea",
+  "filename": "jquery.PrintArea.min.js",
+  "version": "2.4.1",
+  "description": "Prints a specific area of the page. Using json settings, the printed area is either opened in a popup or in a hidden iframe. Minimal setup and easy to use. See the demo for examples and usage of the options.",
+  "keywords": [
+    "jquery",
+    "print",
+    "printing"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RitsC/PrintArea.git"
+  },
+  "authors": [
+    "Chris Ritschard",
+    "Djuri Baars"
```

Comment:
The author is `Chris Ritschard`
https://github.com/RitsC/PrintArea/blob/master/PrintArea.jquery.json#L7
Djuri Baars is not the author.
Thanks

Reply:
@sufuf3 Where do you see PrintArea.css?

Reply:
Here, https://github.com/RitsC/PrintArea/blob/2.4.1/demo/PrintArea.css